### DISCUSSION TITLE
Crossfade to Live Stream

### DISCUSSION THREAD

**Discussion by BornKillaz** [AzuraCast/AzuraCast#7109] [Q&A]
Hi,

Currently, when a DJ connects, there is no crossfade between the track being played and the stream, causing the live stream to start abruptly.

Is there any way to enable a crossfade?

Thank you.

**Comment by unudelaradio** [comment]
I'm also curious about how to make this work. One more reason is that after Live ends, a small part of the last song played in AutoDJ continues to play for a few seconds. A way to make AutoDJ fade out before live starts will be great!

**Comment by Moonbase59** [comment]
> One more reason is that after Live ends, a small part of the last song played in AutoDJ continues to play for a few seconds.

Yep, that really sucks. In all my tries (using sequence, add, fallback, etc.) I couldn’t make that "buffer snippet" go away when coming back from live. All else is fine, but when coming back from live it insists in playing what sounds like the buffered fade-out part of the last song played when starting the next song.

Just re-tested using Rolling Release #[00986ee](https://github.com/AzuraCast/AzuraCast/commit/00986ee3c0d090340d3e23dce085f454d52bc77d) (2024-05-13 5:06).

Unfortunately, my issue https://github.com/AzuraCast/AzuraCast/issues/6933 went stale and was closed.

**Comment by BornKillaz** [comment]
This was a breeze with the Shoutcast Transcoder. Unfortunately, the transcoder was discontinued, although it still works just fine. Azuracast with it would solve this and other issues. 

I hope someone can find a solution for it, as I would like to continue using Azuracast. 

This must be a Liquidsoap issue, right?
